<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>39</y>
     <width>591</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xmax</string>
   </property>
  </widget>
  <widget class="QLabel" name="l4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>rangY:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>xmin</string>
   </property>
  </widget>
  <widget class="QLabel" name="l3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rangX:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="xMin">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-100000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
   <property name="value">
    <double>-10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="xMax">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-100000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rangX">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-10000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
   <property name="value">
    <double>-10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rangY">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-100000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
